src/server/utilities/cspHeader/index.js

```diff
@@ -230,6 +232,10 @@ const directives = {
       'https://www.instagram.com', // Social Embeds
       'http://www.instagram.com', // Social Embeds
       'https://cdn.syndication.twimg.com', // Social Embeds
+      'https://static.stage.bbc.co.uk', // STY include
+      'http://static.stage.bbc.co.uk', // STY include
+      'https://static.bbc.co.uk', // STY include
+      'http://static.bbc.co.uk', // STY include
```

Comment:
Why do we need to add both HTTP and HTTPS? I would have expected any HTTP resources to produce a [mixed content](https://developers.google.com/web/fundamentals/security/prevent-mixed-content/what-is-mixed-content) warning.

Reply:
for the asset we have locally it fails at `http://static.bbc.co.uk` and that on test fails at `https://static.bbc.co.uk`, same with the one on test which we saw failing earlier for `img-src 'http://news.files.bbci.co.uk'` which we don't have but we have `img-src 'https://news.files.bbci.co.uk'`

Reply:
some of the directives in this file also have declaration for variations of `https` and `http`

Reply:
Please can you link to the include that is referencing http resources?

> some of the directives in this file also have declaration for variations of https and http
Yes I saw, and I'm concerned about that too. Having HTTP URLs in the CSP is a potential security hole. I'll take those up separately though.

Reply:
@jamesdonoh here's [the link](https://www.test.bbc.com/gahuza/36575997?renderer_env=live)